src/SubmitPage/SubmitForm.js

```diff
@@ -64,8 +64,8 @@ const emptyData = {
   Locality: "",
   [PLACE_ID]: "",
   [STORE_ADDRESS]: "",
-  [OPENING_TIME]: null,
-  [CLOSING_TIME]: null,
+  [OPENING_TIME]: "",
+  [CLOSING_TIME]: "",
```

Comment:
This changes needs to be reverted. The time picker component doesn't not take a string. It either needs to be a date type object of null. We convert these values to string type while sending it back to backend

Reply:
Did you get an error with `null` as default value

Reply:
I was able to submit the form successfully for a location in Brazil. This change was made by prettier, so I guess we gotta exclude this rule from there. Anyway, that's easily revertible.